clients/client-shell/cmds/version/version.go

```diff
@@ -23,8 +51,63 @@ var (
 
 func init() {
 	root.Command.AddCommand(Command)
+	root.Command.AddCommand(Updcommand)
 }
 
+
 func printVersion(cmd *cobra.Command, _ []string) {
 	fmt.Fprintf(cmd.OutOrStdout(), "taskcluster version %s\n", VersionNumber)
 }
+
+func check(cmd *cobra.Command, _ []string) {
+
+    //Get response and also check for error in  from git's REST API
+
+    response, err := http.Get ("https://api.github.com/repos/taskcluster/taskcluster/releases/latest") 
+
+    if err != nil {
+        fmt.Print(err.Error())
+        
+    }
+
+    // Read the whole response body and check for any errors
+
+
+    s, err := ioutil.ReadAll(response.Body)
+    
+    if err != nil {
+        fmt.Errorf(err.Error())
+        
+    }
+
+    //Create an object for the struct to parse the json data into given structure
+    
+    R := Release{}
+    if err := json.Unmarshal([]byte(s), &R); err != nil {
+        fmt.Errorf(err.Error())
+    }
+
+    //Check if taskcluster is already up to date
+
+	if R.Name == "v"+VersionNumber {
+       
+        fmt.Printf("taskcluster is already updated")
+    
+    
+    } else {
+    if runtime.GOOS == "linux" {
+    fmt.Printf(R.Asslist[1].Download)
```

Comment:
These assets may not always be in this order.  It will be better to find the asset with the matching filename for GOARCH and GOOS.